src/components/dialogs/substation-creation-dialog.js

```diff
@@ -0,0 +1,273 @@
+        >
+            <DialogTitle>
+                {intl.formatMessage({ id: 'CreateSubstation' })}
+            </DialogTitle>
+            <DialogContent>
+                <Grid container spacing={2}>
+                    <Grid item xs={4} align="start">
+                        <TextField
+                            size="small"
+                            fullWidth
+                            id="substation-id"
+                            label={intl.formatMessage({ id: 'ID' })}
+                            variant="filled"
+                            value={substationId}
+                            onChange={handleChangeSubstationId}
+                            /* Ensures no margin for error message (as when variant "filled" is used, a margin seems to be automatically applied to error message
+                               which is not the case when no variant is used) */
+                            FormHelperTextProps={{
+                                className: classes.helperText,
+                            }}
+                            {...(errors.get('substation-id')?.error && {
+                                error: true,
+                                helperText: intl.formatMessage({
+                                    id: errors.get('substation-id')?.errorMsgId,
+                                }),
+                            })}
+                        />
+                    </Grid>
+                    <Grid item xs={4} align="start">
+                        <TextField
+                            size="small"
+                            fullWidth
+                            id="substation-name"
+                            label={
+                                intl.formatMessage({ id: 'Name' }) +
+                                ' ' +
+                                intl.formatMessage({
+                                    id: 'Optional',
+                                })
+                            }
+                            value={substationName}
+                            onChange={handleChangeSubstationName}
+                            variant="filled"
+                        />
+                    </Grid>
+                    <Grid item xs={4} align="start">
+                        <Autocomplete
+                            id="select-countries"
+                            onChange={(event, newValue) => {
+                                handleChangeCountry(newValue);
+                            }}
+                            input={<BootstrapInput />}
+                            options={Object.keys(countriesList.object())}
+                            style={BootstrapInput.input}
+                            getOptionLabel={(code) => countriesList.get(code)}
+                            renderInput={(props) => (
+                                <TextField
+                                    {...props}
+                                    variant="outlined"
```

Comment:
`variant` should be `"filled"` and a property `size="small"` should be added in order for the component to be the same height as other fields in the row.